modules/client/src/erc20/internal/utils.ts

```diff
@@ -78,31 +78,31 @@ export function toErc20Proposal(
       // precision
 
       // minSupport: decodeRatio(
-      //   BigInt(proposal.supportRequiredPct),
+      //   BigInt(proposal.totalSupportThresholdPct),
       //   2,
       // ),
       // minTurnout: decodeRatio(
-      //   BigInt(proposal.participationRequiredPct),
+      //   BigInt(proposal.relativeSupportThresholdPct),
       //   2,
       // ),
       // TODO DELETE ME
       minSupport: parseFloat(
-        formatEther(proposal.supportRequiredPct),
+        formatEther(proposal.totalSupportThresholdPct),
```

Comment:
is this `formatEther()` necessary? to imply that a value labeled with `...Pct` isn't actual a percentage but a number of coins with 18 decimals.

Reply:
I assume that the current plugin implementation expects the percentage to be expressed with the 18 decimals... of token balances? :-/    `57.00000000000000000000`

Otherwise, it makes no sense IMO

Reply:
As far as I know subgraph should store a number with 18 decimals, so to transform it into a decimal between 0 and 1, yes, it's necessary

Reply:
> I assume that the current plugin implementation expects the percentage to be expressed with the 18 decimals... of token balances? :-/ `57.00000000000000000000`
> 
> Otherwise, it makes no sense IMO

Yes when i talked about this with sarkawt

50% =>  5*10e18 (in subgraph) => 0.5 in the UI